konnect-runtime-setup.sh

```diff
@@ -191,7 +207,7 @@ http_res_body() {
 login() {
     log_debug "=> entering login phase"
 
-    ARGS="--cookie-jar ./$KONNECT_HTTP_SESSION_NAME -X POST -d {\"username\":\"$KONNECT_USERNAME\",\"password\":\"$KONNECT_PASSWORD\"} --url $KONNECT_API_URL/kauth/api/v1/authenticate"
+    ARGS="--cookie-jar ./$KONNECT_HTTP_SESSION_NAME -X POST -d {\"username\":\"$KONNECT_USERNAME\",\"password\":\"$KONNECT_PASSWORD\"} --url $HTTP_PROTOCOL://global.$KONNECT_API_HOST/kauth/api/v1/authenticate"
```

Comment:
we added "$HTTP_PROTOCOL" for support local dev use case, but  "$HTTP_PROTOCOL://global.$KONNECT_API_HOST" still breaking the local dev use case.

suggestion:
Add optional flag "$KONNECT_GLOBAL_API_HOST", if not specified, generates the host as "$HTTP_PROTOCOL://global.$KONNECT_API_HOST". if specified, use "$KONNECT_GLOBAL_API_HOST".